app/src/main/res/values/strings.xml

```diff
@@ -413,5 +413,7 @@
     <string name="translate_help_summary">Bring the Open Food Facts database to your language - no technical knowledge required</string>
     <string name="please_check_your_connection">Please check your connection.</string>
     <string name="faq_url" translatable="false">https://world.openfoodfacts.org/faq</string>
+    <string name="advanced_search_url" translatable="false">https://world.openfoodfacts.org/cgi/search.pl</string>
+    <string name="advance_search_title"><![CDATA[Advanced Search & Charts]]></string>
```

Comment:
advanced_search_title

Reply:
@teolemon Sorry for the error . I have corrected the string .